test/RoaveTest/SecurityAdvisories/VersionTest.php

```diff
@@ -270,4 +298,60 @@ public function nonEquivalentVersionProvider() : array
             ['2.0.1.0.0.0', '2.0.2'],
         ];
     }
+
+    public function differentLengthVersionsProvider() : array
+    {
+        return [
+            ['3', '3.0.0', '3.0.0', '3.0.0'],
+            ['2.1.0-beta1', '2.1', '2.1.0-beta1', '2.1.0'],
+        ];
+    }
+
+    public function versionsToStabilizeProvider(): array
+    {
+        return [
+            ['3.0.0', ['3.0.0', null]],
+            ['2.1.0-beta1', ['2.1.0', '-beta1']],
```

Comment:
Please add more entries here (specifically, the stability flags you checked so far)

Reply:
sure, no problem ^^

Reply:
I wont be able to do it today, will try to fix everything tomorrow